keps/prod-readiness/sig-network/3705.yaml

```diff
@@ -6,3 +6,5 @@ alpha:
   approver: "wojtek-t"
 beta:
   approver: "wojtek-t"
+stable:
+  approver: "wojtek-t"
```

Comment:
@danwinship - There is `TODO; do a manual test` in the PRR part of the KEP.

Was the test run? Can you please fill this in?

[Other than that, LGTM]

Reply:
Reading that now, I'm not even sure what I was planning to test, so I filled that in with more of a "why this question doesn't make sense in this KEP" answer.

Reply:
yes - that makes sense - thanks!